### ISSUE TITLE
Feature: Unify usage of simulation / plots with PK-Sim

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/MoBi#267]
MoBi behaves slightly differently regarding usage of simulation and results and I think this can be very confusing for new user (and even for advanced users).

## Differences:
* MoBi creates one set of results for each simulation instead of managing the current chart. This implementation probably has some historical reasons but at any rate, creation of simulation is not that expensive anymore. My suggestion would be to remove the "historical results" feature of MoBi and only have one and one only set of results

* MoBi simulation does not track observed data used in a simulation. So if some observed data are added to a chart and not displayed in the chart (for example after using a chart template), the observed data are gone. A MoBi simulation should track which observed data are used, like in PKSim

* MoBi Simulation should allow for more than one plot at a time, like in PKSim

## Questions

@Yuri05 @UlrichSi 
Why do we have this difference of behavior. Event if historical result may be sometime useful, would the use of different simulations be enough? Especially many simulation would allow with the help of comparison to actually understand what the different between two simulations is, instead of relying on some extra documentation

## Suggestion:

Unify PK-Simand MoBi by using the logic implemented in PK-Sim in MoBi, as this is the logic that most users are familiar with

**Comment by Yuri05** [comment]
Makes sense for me.
@StephanSchaller @PavelBal What do you think?

**Comment by StephanSchaller** [comment]
I agree. The limiting factor with having multiple simulations in MoBi in one project is memory (MoBi models are often quite large). But I guess the same holds true for having stored historical simulation data.
--> 64 bit! ;-)

**Comment by msevestre** [comment]
The size of the simulation is indeed not significant compared to results. This is what takes a lot of space. 64 bit will allow for more ram to be available, but results will also take more space.... How significant will the gain be is hard to say at the moment

**Comment by UlrichSi** [comment]
To my knowledge, MoBi will eventually clean up behind itself. When reloading a saved project, only the latest simulation result of each simulation will be available. To keep older versions, you will need to set up new simulations (cloning a simulation is still a missing feature, but a veteran on the wish list) or move them to comparison windows before they get lost.

I do not exactly recall why we once implemented the different behavior but it might have the advantage of easier creating comparison charts.

**Comment by AndreDlm** [comment]
> - MoBi creates one set of results for each simulation instead of managing the current chart. This implementation probably has some historical reasons but at any rate, creation of simulation is not that expensive anymore. My suggestion would be to remove the "historical results" feature of MoBi and only have one and one only set of results
> - MoBi simulation does not track observed data used in a simulation. So if some observed data are added to a chart and not displayed in the chart (for example after using a chart template), the observed data are gone. A MoBi simulation should track which observed data are used, like in PKSim
> - MoBi Simulation should allow for more than one plot at a time, like in PKSim

@msevestre Any ideas whether this will be implemented at some point in the near future? Would be very helpful 👍

**Comment by PavelBal** [comment]
I think we agreed on this as part of OSMOSES, see https://github.com/Open-Systems-Pharmacology/OSMOSES/issues/10

**Comment by AndreDlm** [comment]
Great to see that it's being taken up in OSMOSES, thanks @PavelBal :)
